standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/PresentationFramework.xml

```diff
@@ -30924,7 +30924,7 @@ To a <see cref="T:System.Uri" /> that is not located at the site of origin, or
       <summary>The values of this dependency property should be saved or restored by journaling processes, or when navigating by Uniform resource identifiers (URIs).</summary>
     </member>
     <member name="F:System.Windows.FrameworkPropertyMetadataOptions.None">
-      <summary>No options are specified; the dependency property uses the default behavior of the Windows Presentation Foundation (WPF) property system.</summary>
+      <summary>No options are specified; the dependency property uses the default behavior of the WPF property system.</summary>
```

Comment:
let's restore the original and I'll investigate 

```suggestion
      <summary>No options are specified; the dependency property uses the default behavior of the Windows Presentation Foundation (WPF) property system.</summary>
```

Reply:
This one was correct from our discussion.